app_core.py

```diff
@@ -23,7 +24,10 @@ def default(self, o):
             return str(o)
         return super().default(o)
 
+csrf = CSRFProtect()
```

Comment:
move this down to where it is used `csrf.init_app(app)`

Reply:
moved above the csrf.init_app(app)